### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by Azd325**

### CODE REVIEW

**Comment by MarcinWieczorek** [comment]
https://github.com/2m/arch-pkgbuild-builder/blob/master/entrypoint.sh#L29
results in
`Error: gitkraken-7.0.1.tar.gz is empty or is not a valid package`

**Comment by Azd325** [comment]
> https://github.com/2m/arch-pkgbuild-builder/blob/master/entrypoint.sh#L29
> results in
> `Error: gitkraken-7.0.1.tar.gz is empty or is not a valid package`

Yeah, that I saw too. This PR is already using a different `entrypoint.sh`.

**Comment by MarcinWieczorek** [comment]
Which has the same bug, so I pointed at upstream, as you didn't introduce it.

**Comment by Azd325** [comment]
> Which has the same bug, so I pointed at upstream, as you didn't introduce it.

Ahh, this I didn't saw. :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (MarcinWieczorek) [inline .github/workflows/ci.yml:1]: What is this line for?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+name: CI
+on: [push]
+
+jobs:
+  validation:
+    runs-on: ubuntu-latest
+    name: Valdiate package
```
Comment (MarcinWieczorek) [inline .github/workflows/ci.yml:8]: typo "Validate"

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Azd325) [reply .github/workflows/ci.yml:1]: [YAML uses three dashes (“---”) to separate directives from document content. This also serves to signal the start of a document if no directives are present. Three dots ( “...”) indicate the end of a document without starting a new one, for use in communication channels.](https://yaml.org/spec/1.2/spec.html#id2760395)

**Pull request merged**